docs/guides/assets.md

```diff
@@ -26,11 +26,11 @@ The **asset base path** is the directory that Stencil will use to resolve assets
 When a component uses an asset, the asset's location is resolved relative to the asset base path.
 
 The asset base path is automatically set for the following output targets:
-- [dist](/distribution)
-- [hydrate](/hydrate-app)
-- [www](/www)
+- [dist](../output-targets/dist.md)
+- [hydrate](./hydrate-app.md)
```

Comment:
Non blocking: This URL is different from what we use in the [v2](https://github.com/ionic-team/stencil-site/pull/1022/files#diff-419f03d30bdf123f841df6e0dfad6010ae1f847f25d3511889bbe212d40f1fbeR30) & [v3.0 docs](https://github.com/ionic-team/stencil-site/pull/1022/files#diff-419f03d30bdf123f841df6e0dfad6010ae1f847f25d3511889bbe212d40f1fbeR30). They resolve to the same place, just a minor difference in their values. Calling this out just in case that causes alarms to go off in your head/there's an issue I'm not seeing.

Reply:
ah yeah I think that's fine, just a side-effect of trying to do these edits quickly haha